adapters/repos/db/shard_replication.go

```diff
@@ -147,7 +156,7 @@ func (s *Shard) prepareDeleteObjects(ctx context.Context, requestID string, docI
 		for i, r := range result {
 			entry := replica.UUID2Error{UUID: string(r.UUID)}
 			if err := r.Err; err != nil {
-				entry.Error = err.Error()
+				entry.Error = replica.Error{Code: replica.StatusConflict, Msg: err.Error()}
```

Comment:
I think I am not clear on what `conflict` is indicating here

Reply:
Same as before